.dockerignore

```diff
@@ -0,0 +1,4 @@
+.git
```

Comment:
@eliasreis54 O que acha de ignorar tudo menos o diretório `/src` e os arquivos `package.json`, `package-lock.json` e `Dockerfile`?
Não sei se é a melhor opção e se continuará funcionando dessa forma.

Referência: https://stackoverflow.com/questions/28097064/dockerignore-ignore-everything-except-a-file-and-the-dockerfile

Reply:
Eu cheguei a pensar nisso, porém, vai que rem algum `.env` futuramente, ou algum outro arquivo de configuração que fique oculto tem que lembrar de não ignorar ele no docker. 

Caso não veja problema, podemos ignorar tudo menos os arquivos necessários.

Reply:
Concordo com você. Acho interessante então adicionar os dirs `/docs`, `/test` e o arquivo `CHANGELOG.md`.

Reply:
Prontinho =)
Conseguimos chegar a 109MB de imagem. To vendo se conseguimos melhorar mais ainda, porém, até agora já ta bem melhor.

Reply:
Eita, isso tudo pra 1 MB kkkkk